README.adoc

```diff
@@ -0,0 +1,389 @@
+= python-domino
+
+This library provides bindings for the https://dominodatalab.github.io/api-docs/[Domino Data Lab API].
```

Comment:
let's also mention that this is a legacy document, and may not reflect the most up today api,
or update link to current public api docs?